apps/qubit/modules/digitalobject/actions/editSubtitlesComponent.class.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Access to Memory (AtoM) software.
+ *
+ * Access to Memory (AtoM) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Access to Memory (AtoM) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Access to Memory (AtoM).  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * digitalObject actions.
+ *
+ * @package    AccesstoMemory
+ * @subpackage digitalObject
+ * 
+ */
+class DigitalObjectEditSubtitlesComponent extends sfComponent
+{
+  public function execute($request)
+  {
+  }
+}
+
```

Comment:
An empty component class usually indicates that this could be directly a partial.

Reply:
Yep, partial call below.

```
<?php echo include_partial('editSubtitles', array('resource' => $resource, 'subtitles' => $videoTrack, 'form' => $form, 'usageId' => $usageId)) ?>
```

Reply:
Just to flag it, same issue here:

https://github.com/artefactual/atom/blob/qa/2.x/apps/qubit/modules/digitalobject/actions/editRepresentationComponent.class.php